src/snowflake/snowpark/mock/_connection.py

```diff
@@ -396,7 +330,6 @@ def _get_string_datum(self, query: str) -> Optional[str]:
     # def get_result_attributes(self, query: str) -> List[Attribute]:
     #     return convert_result_meta_to_attribute(self._cursor.describe(query))
 
-    @_Decorator.log_msg_and_perf_telemetry("Uploading file to stage")
     def upload_file(
```

Comment:
this API is not used in local testing, in live it's used internally for uploading pkgs